optuna/study/_study_summary.py

```diff
@@ -114,7 +114,7 @@ def directions(self) -> Sequence[StudyDirection]:
     def system_attrs(self) -> Dict[str, Any]:
         warnings.warn(
             "`system_attrs` has been deprecated in v3.1.0. "
-            "The removal of this feature is currently scheduled for v6.0.0, "
+            "The removal of this feature is currently scheduled for v5.0.0, "
```

Comment:
Could you also update the following line?

https://github.com/optuna/optuna/blob/b2d6f711b9a0eb0adeb1ee8a4b9fa37b88c2fd31/optuna/study/_study_summary.py#L42-L44

Reply:
Thank you for your review. I fixed it.

Reply:
And I also create PR to optuna-integration (https://github.com/optuna/optuna-integration/pull/4 )